content/en/docs/components/pipelines/v2/local-execution.md

```diff
@@ -129,6 +129,7 @@ Local execution is designed to help quickly *test* components and pipelines loca
 Local execution comes with several limitations:
 - Local execution does not feature optimizations and additional features such as caching, retry, etc. While these feature are important for production pipelines, they are less critical for a local testing environment. You will find that task methods like `.set_retry`, `.set_caching_options`, etc. have no effect locally.
 - Local execution makes simple assumptions about the resources available on your machine. Local execution does not support specifying resource requests/limits/affinities related to memory, cores, accelerators, etc. You will find that task methods like `.set_memory_limit`, `.set_memory_request`, `.set_accelerator_type` etc. have no effect locally.
+- Local execution does not feature authentication mechanisms. If your component interacts with cloud resources or takes other privileged action, your pipeline is better-suited for testing in the cloud.
```

Comment:
Nit (provided my edits preserve the implications of the statement): Local execution doesn't support authentication mechanisms. If your component interacts with cloud resources or requires other privileged actions, you need to test your pipeline in the cloud.